client/src/i18n/en/enterprise.json

```diff
@@ -21,7 +21,9 @@
       "ENABLE_PREPAYMENT_INVOICE_LABEL" : "Enable to see the balance on invoice",
       "ENABLE_PREPAYMENT_INVOICE_HELP_TEXT" : "Enabling this feature allows to see the balance and the paid amount on an invoice",
       "ENABLE_BARCODES_LABEL" : "Enables barcodes throughout the application",
-      "ENABLE_BARCODES_HELP_TEXT" : "Enabling this feature will place barcodes on most printed records and enable options to scan barcodes for document inputs."
+      "ENABLE_BARCODES_HELP_TEXT" : "Enabling this feature will place barcodes on most printed records and enable options to scan barcodes for document inputs.",
+      "ENABLE_AUTO_STOCK_ACCOUNTING_LABEL" : "Enables realtime stock accounting",
+      "ENABLE_AUTO_STOCK_ACCOUNTING_HELP_TEXT" : "Enabling this feature will write into the posting journal in realtime all stock movements transaction. It require that all inventory accounts must be correctly configured"
```

Comment:
This is a really good translation!  I'm impressed :)

I'm not completely sure why, but in English it makes more sense to reverse your phrases a little bit:
> Enabling this feature will write stock movement transactions into the posting journal in real time.  It requires that all inventory accounts are correctly configured.